src/LayerPlan.cpp

```diff
@@ -622,8 +635,9 @@ TimeMaterialEstimates GCodePlanner::computeNaiveTimeEstimates()
     return ret;
 }
 
-void GCodePlanner::processFanSpeedAndMinimalLayerTime()
+void LayerPlan::processFanSpeedAndMinimalLayerTime(Point starting_position)
 {
+    extruder_plans[0].start_position = starting_position;
```

Comment:
Fixed. the `start_position` wasn't actually used. It seemed to only be used by the functions which are called from this function, so I just gave all of those functions an extra parameter.